papers/MeerTRAP/plot_z_comparison.py

```diff
@@ -0,0 +1,248 @@
+""" 
```

Comment:
I love this figure

Reply:
It is beautiful, isn't it? :-) I really like the colour scheme!